components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/dao/RegistrySubscriberDAOImpl.java

```diff
@@ -38,15 +38,18 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.Map;
+import java.util.Properties;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import static org.wso2.carbon.identity.entitlement.PDPConstants.SUBSCRIBER_ID;
+
 /**
  * This implementation handles the subscriber management in the Registry.
  */
 public class RegistrySubscriberDAOImpl implements SubscriberDAO {
 
-    public static final String SUBSCRIBER_ID = "subscriberId";
```

Comment:
why are we removing it

Reply:
It is added to the PDPConstants class instead. Since `RegistrySubscriberDAOImpl.java` class was newly introduced, it is safe to remove this public constant.